Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -3514,6 +3514,7 @@ def __call__(self, root):
         # True for 'cdef func() nogil:' functions, as the GIL may be held while
         # calling this function (thus contained 'nogil' blocks may be valid).
         self.nogil_declarator_only = False
+        self.current_gilstat_node_knows_gil_state = False
```

Comment:
The comment above doesn't apply to this attribute any more, so better separate the two visually.
```suggestion

        self.current_gilstat_node_knows_gil_state = False
```